corehq/apps/cloudcare/static/cloudcare/js/form_entry/webformsession.js

```diff
@@ -454,10 +454,14 @@ WebFormSession.prototype.submitForm = function(form) {
                 form.submitting();
                 self.onsubmit(resp);
             } else {
-                $.each(resp.errors, function(ix, error) {
+                $.each(resp.errors, function (ix, error) {
                     self.serverError(getForIx(form, ix), error);
                 });
-                alert('There are errors in this form; they must be corrected before the form can be submitted.');
+                if (resp.notification) {
```

Comment:
What happens if there is no `resp.notification`? Is it silent where it used to display a (generic) message?

Reply:
good point Danny